app/views/result/_people_and_organisations.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if enabled?("organisations") || enabled?("people")
+  return_array = []
+  %w(people organisations).each do |thing|
+    if enabled?(thing) && side[thing].present?
+      side[thing].each do |person_or_organisation| %>
+        <br><a href="<%= link_format(person_or_organisation['link']) %>"><%= person_or_organisation['title'] %></a>
+      <% end
+    end
+  end
+end %>
```

Comment:
This can and should simplify. And please, don't use multi line bits of logic. they are hard to understand.